File: server/models/event.js
const mongoose = require("mongoose");
const schema = mongoose.Schema;


//create schema and model

const eventSchema = new schema({
    eventName: String,
    provider: String,
    event: {
      timeRelevance: String,
      genre: String,
      place: String,
      friendsGoing: String,
      signedUp: String,          // val used to determine if event was signed up for
      identifier: String         // random String generated, which is used to idetify each event
    }
});

const uModel = mongoose.model("event", eventSchema);

console.log("event model online");
module.exports = eventSchema;

File: Lux/Event.swift
//
//  Event.swift
//  Lux
//
//  Created by Dave Jha on 11/23/18.
//  Copyright © 2018 Dave Jha. All rights reserved.
//

import Foundation

struct Event: Codable {
    
    var eventName: String?
    var provider: String?
    var timeRelevance: String?
    var genre: String?
    var place: String?
    var friendsGoing: [String]?
    
}
